dev/com.ibm.ws.jpa_30_fat/test-applications/jpabootstrap/resources/jpa-3.0/WEB-INF/classes/META-INF/persistence.xml

```diff
@@ -0,0 +1,11 @@
+<persistence xmlns="https://jakarta.ee/xml/ns/persistence"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd"
+        version="3.0">
+    <persistence-unit name="JPAPU">
+    	<properties>
+			<!-- EclipseLink should create the database schema automatically -->
+			<property name="eclipselink.ddl-generation" value="drop-and-create-tables" />
```

Comment:
Don't we use DDL generation? If this is necessary, can we change to: 
```
<property name="jakarta.persistence.schema-generation.database.action" value="drop-and-create"/>
```

Reply:
Don't think we have that set up for this test, a todo for the future.